README.md

```diff
@@ -40,10 +44,11 @@ What is included in the software unit-test? What is not? Fill this table.
 |---------------------------|---------------|---
 Battery Data-accuracy       | No            | We do not test the accuracy of data
 Computation of maximum      | Yes           | This is part of the software being developed
-Off-the-shelf PDF converter | _enter Yes/No | _enter reasoning
-Counting the breaches       | _enter Yes/No | _enter reasoning
-Detecting trends            | _enter Yes/No | _enter reasoning
-Notification utility        | _enter Yes/No | _enter reasoning
+Computation of minimum      | Yes           | This is part of the software being developed
+Off-the-shelf PDF converter | Yes 	    | PDF is among the deliverables of the software being developed 
```

Comment:
" Libraries to convert captured data to PDF" will you write unit test code for imported library?